src/views/ev3/ev3.jsx

```diff
@@ -13,328 +13,201 @@ const Page = require('../../components/page/www/page.jsx');
-                                className="project-card"
-                                href="https://downloads.scratch.mit.edu/ev3/ev3-wave-hello.sb3"
-                            >
-                                <div className="project-card-image">
-                                    <img src="/images/ev3/starter-wave-hello.png" />
-                                </div>
-                                <div className="project-card-info">
-                                    <h4>Wave Hello</h4>
-                                    <p>
-                                        Make a puppet robot and have a friendly chat.
-                                    </p>
-                                </div>
-                            </a>
-                            <a
-                                download
-                                className="project-card"
-                                href="https://downloads.scratch.mit.edu/ev3/ev3-distance-instrument.sb3"
-                            >
-                                <div className="project-card-image">
-                                    <img src="/images/ev3/starter-distance-instrument.png" />
-                                </div>
-                                <div className="project-card-info">
-                                    <h4>Distance Instrument</h4>
-                                    <p>
-                                        Move your body in front of the sensor to make music.
-                                    </p>
-                                </div>
-                            </a>
-                            <a
-                                download
-                                className="project-card"
-                                href="https://downloads.scratch.mit.edu/ev3/ev3-space-tacos.sb3"
-                            >
-                                <div className="project-card-image">
-                                    <img src="/images/ev3/starter-flying-game.png" />
-                                </div>
-                                <div className="project-card-info">
-                                    <h4>Space Tacos</h4>
-                                    <p>
-                                        Build your own controller to catch tacos in space.
-                                    </p>
-                                </div>
-                            </a>
-                        </FlexRow>
-                    </FlexRow>
-                </div>
-                <div className="faq inner">
+                        </Step>
+                    </Steps>
+                    <hr />
+                    <h3>Starter Projects</h3>
+                    <Steps>
+                        <ProjectCard
+                            cardUrl="https://downloads.scratch.mit.edu/ev3/ev3-wave-hello.sb3"
+                            description="Make a puppet robot and have a friendly chat."
```

Comment:
The project cards should also be translatable.

Reply:
The l10n issues are being tracked in #1996 - should they be fixed in this PR or in a separate PR for that issue?

Reply:
Sorry - I didn't see that one. It should be a separate PR!